<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPaper</class>
 <widget class="QMainWindow" name="TestPaper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestPaper</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>570</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>561</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Create Qustion</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Create MCQ</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Question</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="msqQuestion">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>731</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>select No of Choise</string>
       </property>
      </widget>
      <widget class="QComboBox" name="noOfChoice">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="answer1lable">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>answer 1</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="answer1">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="answer2lable">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>130</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>answer 2</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="answer2">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>120</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="answer3lable">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>answer 3</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="answer3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="answer4lable">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>170</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>answer 4</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="answer4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>160</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="answer5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>200</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="answer5lable">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>answer 5</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="answer6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>200</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="answer6lable">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>210</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>answer 6</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addMcq">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>280</y>
         <width>271</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>add qustion</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>No of answers per quseion</string>
       </property>
      </widget>
      <widget class="QComboBox" name="noOfAnswers">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>250</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Single</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Multiple</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>841</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Create Essay quseion</string>
      </property>
      <widget class="QTextEdit" name="essayQusion">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>731</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Question</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addEssay">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>271</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>add qustion</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>View Creating Paper</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>TestPaper</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>1011</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
